jwst/barshadow/bar_shadow.py

```diff
@@ -105,6 +105,7 @@ def do_correction(input_model, barshadow_model):
                 slitlet.data /= correction
```

Comment:
The previous few lines are comments, and the one that refers to var_rnoise should be updated.

Reply:
Good catch. I've updated it.